Gemfile

```diff
@@ -14,7 +16,16 @@ group :development, :test do
   gem "jeweler", "~> 1.8.4"
   gem 'pry'
   gem 'pry-nav'
-  gem 'pry-stack_explorer'
+  gem 'pry-stack_explorer', platform: :ruby
   gem 'webmock'
   gem 'sinatra'
 end
+
+@@check ||= at_exit do
+  begin
+    require 'lock_jar/bundler'
```

Comment:
Hmm, can you add an explicit check for JRuby and a one-line comment explaining what it does? It would make the intent of the code clearer.